platform/machine/brightbox/cluster.go

```diff
@@ -0,0 +1,80 @@
+// Copyright The Mantle Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package brightbox
+
+import (
+	"crypto/rand"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/flatcar/mantle/platform"
+	"github.com/flatcar/mantle/platform/conf"
+)
+
+type cluster struct {
+	*platform.BaseCluster
+	flight *flight
+}
+
+func (bc *cluster) NewMachine(userdata *conf.UserData) (platform.Machine, error) {
+	conf, err := bc.RenderUserData(userdata, map[string]string{
+		"$public_ipv4":  "${COREOS_OPENSTACK_IPV4_PUBLIC}",
+		"$private_ipv4": "${COREOS_OPENSTACK_IPV4_LOCAL}",
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	var keyname string
+	if !bc.RuntimeConf().NoSSHKeyInMetadata {
+		keyname = bc.flight.Name()
+	}
+	instance, err := bc.flight.api.CreateServer(bc.vmname(), keyname, conf.String())
```

Comment:
```suggestion
	instance, err := bc.flight.api.CreateServer(bc.vmname(), conf.String())
```
It's ok to not support ssh keys in metadata (same as in VMware ESXi or qemu)